tests/test_urbanopt_des_results.py

```diff
@@ -0,0 +1,138 @@
+import shutil
+import unittest
+import warnings
+from pathlib import Path
+
+from urbanopt_des.urbanopt_analysis import URBANoptAnalysis
+from urbanopt_des.urbanopt_geojson import DESGeoJSON as URBANoptGeoJSON
+
+# suppress some warnings -- mostly from pandas
+warnings.filterwarnings("ignore", category=RuntimeWarning)
+warnings.filterwarnings("ignore", category=DeprecationWarning)
+warnings.filterwarnings("ignore", category=UserWarning)
+warnings.simplefilter(action="ignore", category=FutureWarning)
+
+
+class UrbanoptDesResultsTest(unittest.TestCase):
+    # More comprehensive test with 3 buildings and a 5G district energy system.
+    # This file requires the OpenStudio/EnergyPlus results
+    # for each building to be present in the tests/data/three_building_5G directory.
+    def setUp(self):
+        self.data_dir = Path(__file__).parent / "data" / "three_building_5G"
+
+        # if the three"building_test output directory exists then delete it
+        if (self.data_dir / "three_building_test" / "output").exists():
+            shutil.rmtree(self.data_dir / "three_building_test" / "output")
+
+        # delete the modelica_variables.json in any subfolder
+        for path in (self.data_dir / "three_building_test_des_agg").rglob("modelica_variables.json"):
+            if path.is_file():
+                path.unlink()
+
+    def test_valid_results(self):
```

Comment:
this is the work horse... lots of sequential command calls to build up the entire results